hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -926,8 +926,12 @@ static RecordMergeMode inferRecordMergeModeFromMergeStrategyId(String recordMerg
     }
   }
 
-  public String getPreCombineField() {
-    return getString(PRECOMBINE_FIELD);
+  public Option<List<String>> getPreCombineFieldList() {
```

Comment:
I think we should swap the naming here. We want developers to be aware that there are potentially multiple strings as ordering fields so we should make this `getPrecombineFields` and the other method `getPrecombineFieldsAsString`

Reply:
Addressed. Also renamed `PRECOMBINE_FIELD` to `PRECOMBINE_FIELDS`